src/visual.ts

```diff
@@ -680,6 +680,8 @@ module powerbi.extensibility.visual {
                 return;
             }
 
+            let dataViewValues: any = visualUpdateOptions.dataViews[0].categorical.values[0];
```

Comment:
It'd be better to allow users to control display units and precision via formatting panel.

Reply:
It's unnecessary line. I forgot to remove it.

Reply:
Sorry. There's no need to implement this for tooltips.